### PR TITLE
Add /crossevolve and move /om to othermetas.js

### PR DESCRIPTION
**Pull request by Spandamn**
Tested, and wont crash.

### CODE REVIEW

File: chat-plugins/othermetas.js [hunk 1]
```diff
@@ -2,6 +2,123 @@
...
+			return this.sendReply('|raw|<center>' + buffer + '</center>');
+		}
+		if (showMonthly) {
+			this.target = 'omofthemonth';
+			this.run('formathelp');
+		} else {
+			this.run('formathelp');
+		}
+	},
+	othermetashelp: ["/om - Provides links to information on the Other Metagames.",
```
Comment (panpawn) [inline chat-plugins/othermetas.js]: Multiline help commands should go like this:
```
othermetashelp: [
     "/om - Provides links to information on the Other Metagames.",
     !om - Show everyone that information. Requires: + % @ * # & ~"
],
```

(substitute the 5 spaces indent with a tab, of course)

File: chat-plugins/othermetas.js [hunk 2]
```diff
@@ -2,6 +2,123 @@
...
+		} else if (weightkg >= 10) {
+			gnbp = 40;
+		} // Aah, only if `template` had a `bst` property.
+		let bst = stats['hp'] + stats['atk'] + stats['def'] + stats['spa'] + stats['spd'] + stats['spe'];
+		let text = `<b>${poke1.species}</b> ===> <b>${poke2.species}</b>:<br />`;
+		text = `${text}<b>Stats</b>: ${Object.values(stats).join('/')}<br />`;
+		text = `${text}<b>BST</b>: ${bst}<br />`;
+		text = `${text}<b>Type:</b> ${type}<br />`;
+		text = `${text}<b>Abilities</b>: ${ability}<br />`;
+		text = `${text}<b>Weight</b>: ${weightkg} kg (${gnbp} BP)`;
```
Comment (panpawn) [inline chat-plugins/othermetas.js]: Why not just do ``text +=`` instead?

File: chat-plugins/othermetas.js [hunk 3]
```diff
@@ -2,6 +2,123 @@
...
+			this.target = 'omofthemonth';
+			this.run('formathelp');
+		} else {
+			this.run('formathelp');
+		}
+	},
+	othermetashelp: ["/om - Provides links to information on the Other Metagames.",
+		"!om - Show everyone that information. Requires: + % @ * # & ~"],
+
+	ce: "crossevolve",
```
Comment (panpawn) [inline chat-plugins/othermetas.js]: you could probably enable this in PMs

File: chat-plugins/othermetas.js (hunk 2 above)
Comment (Zarel) [inline chat-plugins/othermetas.js]: Yeah, I prefer that for readability reasons.

**Comment by Spandamn** [comment]
I'll make these changes along with beautifying the the display:
crossevo
Something like this (like @TheImmortal suggested).
I'll first move /data's HTML part to the server side though.

**Comment by MacChaeger** [comment]
Now seems like a good time to get this through.

**Comment by Zarel** [comment]
@TheImmortal is this still useful?

**Comment by Zarel** [comment]
@Spandamn?

**Comment by TheImmortal** [comment]
Well, no, it isn't on the main server.

**Pull request closed without merging**